pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -423,6 +434,37 @@ func (c *Reconciler) reconcileMetric(ctx context.Context, pa *pav1alpha1.PodAuto
 	return nil
 }
 
+func (c *Reconciler) checkContainerHealth(ctx context.Context, pa *pav1alpha1.PodAutoscaler) error {
```

Comment:
We'll want to rebase to pick up https://github.com/knative/serving/pull/4191 before moving this over to the pod autoscaler.

Reply:
rebased.